messages.go

```diff
@@ -4,22 +4,27 @@ const (
 	uspControlMessageACK       = "ack"
 	uspControlMessageBACKOFF   = "bo"
 	uspControlMessageRECONNECT = "re"
+	uspControlMessageERROR     = "error"
```

Comment:
I noticed we don't do anything for error messages?

Reply:
Good catch, I had something for it but it got lost in a merge. Will add.